tests/operators/test_local.py

```diff
@@ -504,12 +503,94 @@ def test_run_operator_dataset_inlets_and_outlets_airflow_210(caplog):
             install_deps=True,
             append_env=True,
         )
-        seed_operator >> run_operator >> test_operator
+        seed_operator > run_operator >> test_operator
```

Comment:
```suggestion
        seed_operator >> run_operator >> test_operator
```
looks unintentional?

Reply:
Fixed!